ui/src/app/utils/index.tsx

```diff
@@ -463,3 +463,18 @@ export function highlightSearchTerm(searchTerm: string, stringToHighlight: strin
       {word}
     </span>);
 }
+
+const BACKOFF_DEFAULT_WAIT_TIME_IN_MS = 1000;
```

Comment:
What's the plan for testing this retry code? Do we need a test api that can timeout after a configurable amount of time?

Reply:
I had been thinking about that, and proposed something, but people didn't like it. In this case I just manually tested, but creating some sort of unit test for it would be good. I'll see what I can do.